examples/llama2/notebook/llama2/config.json

```diff
@@ -107,6 +107,7 @@
                             }
                         }
                     ],
+                    "data_config": "tiny_codes_train",
```

Comment:
this is not necessary since the evaluator uses a dataloader_func. Also not related to this PR since the input model doesn't have a "dataset"

Reply:
Fixed.